.github/workflows/python-package.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Install
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  venv:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
@davidkhala We already use `nox` to run tests under different Python versions:
- https://github.com/google/dwh-migration-tools/blob/main/client/noxfile.py#L52
- https://github.com/google/dwh-migration-tools/blob/main/client/CONTRIBUTING.md#test

Also, we have formatting, linting and type checking:
- https://github.com/google/dwh-migration-tools/blob/main/client/CONTRIBUTING.md#test

So far we have just been running everything manually. Is it possible to take advantage of that here? @paolomorandini does it matter whether these checks are run via GH actions or our existing kokoro scripts?

Reply:
I think we can use GH actions as long as we don't need secrets.

Reply:
Integration/load tests will need secrets.

Reply:
@ajwelch4 agree, will take a look to see if we can reuse those existing test script into GH workflow.